.gitignore

```diff
@@ -29,3 +29,8 @@ local.properties
 # some runtime files
 .attach*
 
+# some helper scripts copied into the repo root dir
```

Comment:
No, sorry, but it should be exactly here. The whole idea of that is to avoid extra typing of any subdirectories on the command line. I.e. I enter just `./compile.sh` and that's all (actually entering `.` `/` `c` `[TAB]` `[ENTER]`)

Reply:
As long as it's gitignored and it helps Ivan somehow, I think it's fine. It would be cleaner if they were all in one place, but if his workflow depends on having them very accessible, then for everyone else it's just a line in gitignore. It feels messy to have various scripts around but no real downside if it's not checked in.

Reply:
Also, a bit more gentle please with the style of communication :laughing: ... everybody on our team can listen pretty well to reasonable arguments, the way you worded it sounded quite abrasive.

Reply:
I will try

Reply:
@IvanPizhenko Got it. No prob. We can leave it as is. Marked as resolved